src/components/Visualisers/Blockies.tsx

```diff
@@ -0,0 +1,17 @@
+import Blockies from 'react-blockies';
+
+const Block = (props: any) => {
+  return (
+    <Blockies
+      seed={props.address}
+      size={12}
+      scale={3}
+      color="#dfe"
```

Comment:
I think we only need seed, size and scale here. Color props will make it look different from what you can see in metamask for example.